chapters/audio-intelligence/pages/translation.mdx

```diff
@@ -58,7 +58,7 @@ async function pollForResult(resultUrl: string, headers: any) {
 }
 
 async function startTranscription() {
-  const gladiaKey = "YOUR_GLADIA_API_TOKEN";
+  const gladiaKey = "YOUR_GLADIA_API_KEY";
```

Comment:
Could we also update the headers to be `X-Gladia-Key` to match proper capitalization of `X-` custom headers?